spyder/utils/introspection/utils.py

```diff
@@ -211,26 +211,25 @@ def get_keywords(lexer):
     return keywords
 
 
-def get_words_file(file_in):
+def get_words_file(file_path=None, content=None):
```

Comment:
I think we will need an extra keyword param called extension, otherwise how can we tell the extension if only content was provided?